packages/ws-client/src/data-source.ts

```diff
@@ -139,7 +145,7 @@ export function createWSDataSource(wsUrl: string): DataSource {
 
   const promises = new Map<string, [PromiseHandler, PromiseHandler]>()
 
-  const dataSource: DataSource = {
```

Comment:
one reason I prefer to do it this way is that the type error will appear in a more helpful place when you try to assign it to the `DataSource` type immediately

Reply:
Correct! Will revert back this change, you're right!

Reply:
Going to leave it as is for now, but feel free to change later. From the screenshot attached, both versions seem to provide same level of DX.

<img width="831" alt="Screenshot 2024-05-01 at 13 25 46" src="https://github.com/final-ui/final/assets/2464966/6e938b60-97c4-4e92-98c6-c22e180a5c68">
<img width="814" alt="Screenshot 2024-05-01 at 13 25 19" src="https://github.com/final-ui/final/assets/2464966/b410fd7f-53e5-449b-a5bf-7c3166860ac6">